fyle_accounting_mappings/models.py

```diff
@@ -195,14 +195,11 @@ def bulk_create_or_update_expense_attributes(
                     )
                 )
             else:
-                if update and (
-                        ('detail' in attribute and attribute['detail'] != primary_key_map[attribute['value']]['detail'])
```

Comment:
I don't think we need these conditions, we can just update all
The bulk queries aren't that expensive